### DISCUSSION TITLE
Resolving the path of the exe files and not it instance in appdata temp

### DISCUSSION THREAD

**Discussion by VincentHk6** [brentvollebregt/auto-py-to-exe#394] [Q&A]
Dear,

I have the following lines in my .py that i convert to an exe one file window based;

source_path = Path(__file__).resolve()
source_dir = source_path.parent

I use the pass to access files as config files, so i just need to keep those files in the same folder as the exe. 
The problem is that once running the exe it resolve a path here d\\AppData\\Local\\Temp\\_MEI209402\\ .

Why is that, what did i miss to avoid this issue?
Best regards
Vincent

**Comment by brentvollebregt** [comment]
This has been replied to in https://github.com/brentvollebregt/auto-py-to-exe/issues/397

> In the help post, I discuss ["The Difference Between One Directory and One File"](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#the-difference-between-one-directory-and-one-file).
>
>Files are put in a directory like this when using one-file mode. To not have your files moved into a directory like this, use one-directory mode.
>
>When using one-file, a bootloader is needed to start the exe - [more details on this in the PyInstaller docs here](https://pyinstaller.org/en/stable/advanced-topics.html?highlight=_mei#bootloader).

I'll close that issue as it would be more ideal to discuss this here - as it is not an issue with the tool itself.

**Reply by VincentHk6** [reply]
Duly noted and many thanks . 
So the way  the exe is unpacking / unzipping once launched is operating is different than in some earlier version of auto-py-to-exe. Because I had done some exe a few year back and I was able to use the exe file path to access files that I would leave in the same directory as .exe. Is it ?

**Reply by brentvollebregt** [reply]
Na, it has always functioned this way. 

Potentially when you were previously using it, you were using the one-directory mode and not one-file? Or maybe you were using the [one-file resource wrapper](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#the-one-file-resource-wrapper) which would mitigate most path differences in one-file mode.

**Reply by VincentHk6** [reply]
I was using one file , with cmd display and the way I was getting the path is exactly these and nothing more :
source_path = Path(file).resolve()
source_dir = str(source_path.parent)
